packages/tablev2/style.ts

```diff
@@ -0,0 +1,85 @@
+import * as dt from "../design-tokens/build/js/designTokens";
+import { css, cx } from "emotion";
+import { pseudoElTriangle } from "../shared/styles/styleUtils";
+
+export const headerCss = css`
+  box-sizing: border-box;
+  border-top: 1px solid ${dt.themeBorderHeavy};
+  border-bottom: 1px solid ${dt.themeBorderHeavy};
+`;
+
+export const table = (
+  gridTemplateFragments: string[],
+  firstColSticky: boolean = false
+) => css`
```

Comment:
We might want to set the row height with `grid-auto-rows`. By default, the rows will stretch (or shrink) to fill the height of their table parent element.

Reply:
that explains the trouble i had today! thanks a lot!